presto-native-execution/presto_cpp/main/types/PrestoToVeloxSplit.cpp

```diff
@@ -174,20 +33,19 @@ velox::exec::Split toVeloxSplit(
             remoteSplit->location.location),
         splitGroupId);
   }
-  if (auto tpchSplit = std::dynamic_pointer_cast<const protocol::TpchSplit>(
-          connectorSplit)) {
-    return velox::exec::Split(
-        std::make_shared<connector::tpch::TpchConnectorSplit>(
-            scheduledSplit.split.connectorId,
-            tpchSplit->totalParts,
-            tpchSplit->partNumber),
-        splitGroupId);
-  }
+
   if (std::dynamic_pointer_cast<const protocol::EmptySplit>(connectorSplit)) {
     // We return NULL for empty splits to signal to do nothing.
     return velox::exec::Split(nullptr, splitGroupId);
   }
 
+  for (auto& connector : getPrestoToVeloxConnectors()) {
```

Comment:
Could the Split _type() be used to retrieve the PrestoToVeloxConnector that can be used for the split conversion ?